### PR TITLE
SingUp

### PR DESCRIPTION
**Pull request by nguyenvanson332000**
## Checklist
- [ ] This is bug/feature (delete incorrect one)
- [ ] Title including ticket number and title

## What this PR do?
- Rails g controller User setup routes SignUp, validate model User, callback build SignUp Controller and fix view css Static_pages

## Context/Background
More information on the issue that you want to provide, related issues and PRs you have to write information to ask orally or in informal place too.

## Screenshots / Demo & Instruction for testing (if applicable)
image

## References
- Blog posts / web pages relevant to this issue (which you get your solution / idea from and it may help others to understand the issue quickly)
- Do we need update our knowledge base / project wiki?

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -6,6 +6,8 @@
         <li><%= link_to t(".home"), root_path %></li>
         <li><%= link_to t(".exams"), "#" %></li>
         <li><%= link_to t(".login"), "#" %></li>
+        <li><%= link_to "English", locale:"en" %></li>
```
Comment (rikai-hai) [inline app/views/layouts/_header.html.erb]:
```suggestion
        <li><%= link_to "English", locale: "en" %></li>
```

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -6,6 +6,8 @@
...
+        <li><%= link_to "Vietnamese", locale:"vi" %></li>
```
Comment (rikai-hai) [inline app/views/layouts/_header.html.erb]: tương tự

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+      <!DOCTYPE html>
```
Comment (rikai-hai) [inline app/views/layouts/application.html.erb]: ?

File: app/views/static_pages/home.html.slim [hunk 4]
```diff
@@ -1,4 +1,5 @@
 div class="center jumbotron home"
   h1 = t ".header"
   h2 = t ".content"
-  =link_to t(".signup"), "#", class: "btn btn-lg btn-primary"
+  =link_to t(".signup"), signup_path, class: "btn btn-lg btn-primary"
```
Comment (rikai-hai) [inline app/views/static_pages/home.html.slim]: space

Comment (nguyenvanson332000) [reply app/views/static_pages/home.html.slim]: dạ okay anh

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (nguyenvanson332000) [reply app/views/layouts/application.html.erb]: à cái này do format erb để em chỉnh lại

**Review by rikai-hai — APPROVED** [review]

**Pull request merged**